test/algo/nas/test_space_hub_oneshot.py

```diff
@@ -70,6 +70,10 @@ def _hub_factory(alias):
 
 
 def _strategy_factory(alias, space_type):
+    # Autoformer search space require specific hooks
+    if alias == 'random' and space_type == 'autoformer':
```

Comment:
I think `enas` can be supported without effort. `darts` and `gumbel` can be supported with a few lines of changes of the slice.